simde/arm/neon/ld2_lane.h

```diff
@@ -285,10 +285,10 @@ simde_int16x8x2_t simde_vld2q_lane_s16(int16_t const ptr[HEDLEY_ARRAY_PARAM(2)],
   }
   return r;
 }
-#if defined(SIMDE_ARM_NEON_A64V8_NATIVE)
+#if defined(SIMDE_ARM_NEON_A64V8_NATIVE) || defined(SIMDE_ARM_NEON_A32V7_NATIVE)
```

Comment:
```suggestion
#if defined(SIMDE_ARM_NEON_A32V7_NATIVE)
```

If `SIMDE_ARM_NEON_A64V8_NATIVE` is defined, then `SIMDE_ARM_NEON_A32V8_NATIVE` is automaticall defined

https://github.com/simd-everywhere/simde/blob/fee149ae76155f05fbae921884afe68260f2ff83/simde/simde-features.h#L347-L349

and if `SIMDE_ARM_NEON_A32V8_NATIVE` is defined, then `SIMDE_ARM_NEON_A32V7_NATIVE` is automatically defined

https://github.com/simd-everywhere/simde/blob/fee149ae76155f05fbae921884afe68260f2ff83/simde/simde-features.h#L359-L361

So we can simplify the check here and elsewhere in your PR; likewise for `SIMDE_ARM_NEON_A32V7_ENABLE_NATIVE_ALIASES`

Reply:
correct, no need for the `||` (or) , just go for the lowest ARM version requirement